tests/neg/missing-implicit1.check

```diff
@@ -0,0 +1,36 @@
+-- Error: tests/neg/missing-implicit1.scala:16:4 -----------------------------------------------------------------------
+16 |  ff // error
+   |    ^
+   |no implicit argument of type testObjectInstance.Zip[Option] was found for parameter xs of method ff in object testObjectInstance
+   |
+   |The following import might fix the problem:
+   |
+   |  import testObjectInstance.instances.zipOption
+   |         
+-- [E008] Member Not Found Error: tests/neg/missing-implicit1.scala:18:16 ----------------------------------------------
+18 |  List(1, 2, 3).traverse(x => Option(x)) // error
+   |  ^^^^^^^^^^^^^^^^^^^^^^
+   |  value traverse is not a member of List[Int] - did you mean List[Int].reverse?
+   |
+   |  The following import might fix the problem:
+   |
+   |    import testObjectInstance.instances.traverseList
```

Comment:
My next PR will address this problem. It still lists only the single import needed for the toplevel given instance, but if that is not enough will state that the imports "make progress towards fixing the problem" rather than "fix the problem". If the user that adds the import, they will see
the next necessary import in the next compilation run.